src/pages/RepoPage/BundlesTab/BundleContent/constants.ts

```diff
@@ -0,0 +1,35 @@
+export const BundleReportGroups = {
+  JavaScript: 'JAVASCRIPT',
+  CSS: 'STYLESHEET',
+  Fonts: 'FONT',
+  Images: 'IMAGE',
+  Other: 'UNKNOWN',
+} as const
+
+export type BundleReportTypes = keyof typeof BundleReportGroups
+
+export type BundleReportTypeEnums =
+  (typeof BundleReportGroups)[keyof typeof BundleReportGroups]
+
+const REPORT_BUNDLE_MEASUREMENT_SIZE_MAPPING = 'REPORT_SIZE' as const
```

Comment:
Is this all types? If so, updating the variable names to reflect that would make it more clear.

Reply:
Hrrm, personally I'd rather keep it inline with the enum value we're passing. I've refactored it a little bit to read a bit better but I'll also add in a comment explaining what it is.